h2/src/main/org/h2/jdbc/JdbcResultSet.java

```diff
@@ -3459,11 +3463,15 @@ public void updateNClob(String columnLabel, Reader x, long length)
     }
 
     /**
-     * [Not supported]
+     * Updates a column in the current or insert row.
+     *
+     * @param columnLabel the column label
+     * @param x the value
+     * @throws SQLException if the result set is closed or not updatable
      */
     @Override
     public void updateNClob(String columnLabel, NClob x) throws SQLException {
-        throw unsupported("NClob");
+        updateClob(columnLabel, x);
```

Comment:
do we have any tests for this?

Reply:
I removed tests for unsupported exception, but forget to add some normal tests. I just pushed trivial tests for both `updateNClob()` methods that were not implemented before. More advanced tests are not really needed here because CLOB and NCLOB are the same thing in H2.